src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -194,23 +194,16 @@ static LSP.Location ConvertTextSpanWithTextToLocation(TextSpan span, SourceText
         }
 
         public static LSP.DiagnosticSeverity DiagnosticSeverityToLspDiagnositcSeverity(DiagnosticSeverity severity)
-        {
-            switch (severity)
+            => severity switch
             {
-                // TO-DO: Add new LSP diagnostic severity for hidden diagnostics
-                // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1063158
-                case DiagnosticSeverity.Hidden:
-                    return LSP.DiagnosticSeverity.Hint;
-                case DiagnosticSeverity.Info:
-                    return LSP.DiagnosticSeverity.Hint;
-                case DiagnosticSeverity.Warning:
-                    return LSP.DiagnosticSeverity.Warning;
-                case DiagnosticSeverity.Error:
-                    return LSP.DiagnosticSeverity.Error;
-                default:
-                    throw ExceptionUtilities.UnexpectedValue(severity);
-            }
-        }
+                // Hidden is translated in AbstractPullDiagnosticHandler.ConvertTags to pass along appropriate _ms tags
```

Comment:
Maybe good to just unify where these two live - either both in here or both in AbstractPullDiagnosticHandler

Reply:
will do.